R/manage.R

```diff
@@ -762,7 +840,13 @@ download_json_cached <- function(url, update_cached = TRUE) {
         headers <- curl_fetch_headers(url)
         server_last_modified <- req_headers_last_modified(headers)
 
-        length(server_last_modified) == 1 && local_mtime < server_last_modified
+        if (!is.null(server_last_modified)) {
+          length(server_last_modified) == 1 &&
+            local_mtime < server_last_modified
+        } else {
+          # otherwise cache for 10 minutes
+          (local_mtime + 60 * 10) < Sys.time()
```

Comment:
Seems fast for me. But we can keep it. 

Thinking of at least 1 hr.  8 hrs? 24hrs?

Reply:
`{rlang}` 's `.frequency` defaults to 8 hrs. https://rlang.r-lib.org/reference/abort.html#arg--frequency

I feel that's pretty good. Otherwise I'm waiting to download chrome multiple times a day.